static/app/views/alerts/issueRuleEditor/sentryAppRuleModal.tsx

```diff
@@ -0,0 +1,45 @@
+import {Fragment} from 'react';
+
+import {closeModal, ModalRenderProps} from 'app/actionCreators/modal';
+import {tct} from 'app/locale';
+import SentryAppExternalForm, {
+  Config,
+} from 'app/views/organizationIntegrations/sentryAppExternalForm';
+
+type Props = ModalRenderProps & {
+  sentryAppInstallationUuid: string;
+  appName: string;
+  config: Config;
+  resetValues: {[key: string]: any};
+  onSubmitSuccess: Function;
```

Comment:
Can this be `React.ComponentProps<typeof SentryAppExternalForm>['onSubmitSuccess']` ?

Reply:
yep, made the change didn't know you can access types like that 👍